static/game/deathscreen.js

```diff
@@ -0,0 +1,83 @@
+import { Vector } from "../modules/vector.js";
+import { GuiElement } from "../modules/guielement.js";
+import { centeredText } from "./draw.js";
+import { getScreenDimensions } from "../modules/gamemanager.js";
+import { getImportantEntity } from "../modules/gamemanager.js";
+
+/**
+ * @param {number} num
+ * @return {string}
+ */
+const toScoreString = num => {
+  let str = Math.floor(num).toString();
+  str = str.padStart(6, "0");
+  return str;
+};
```

Comment:
Instead of copying this from ScoreDisplay, just export it from there and import it here